src/extension-test/integration/suite/extension-test.ts

```diff
@@ -23,12 +27,80 @@ suite('Extension Test Suite', () => {
     //   context.workspaceState.get('selectedCljTypeName')
     // });
 
-    test('open a file and close it again, w/o croaking', async () => {
-        expect(vscode.window.activeTextEditor).undefined;
-        const testClj = await openFile(path.join(util.testDataDir, 'test.clj'));
-        vscode.commands.executeCommand('workbench.action.closeActiveEditor');
+    test('Sample test', async () => {
         await sleep(1000);
-        expect(vscode.window.activeTextEditor).undefined;
+        assert.equal(-1, [1, 2, 3].indexOf(5));
+        assert.equal(-1, [1, 2, 3].indexOf(0));
+    });
+
+    // test("open a file and close it again, w/o croaking", async () => {
+    //   expect(vscode.window.activeTextEditor).undefined;
+    //   const testClj = await openFile(path.join(util.testDataDir, 'test.clj'));
+    //   vscode.commands.executeCommand("workbench.action.closeActiveEditor");
+    //   await sleep(1000);
+    //   expect(vscode.window.activeTextEditor).undefined;
+    // });
+
+    test('connect to repl', async function () {
+        console.log('connect to repl');
```

Comment:
This probably should be _'start repl and connect (jack-in)'_ as _connect_ has a the special meaning in Calva of assuming the REPL is already started.